package.json

```diff
@@ -7,28 +7,27 @@
     "test": "test"
   },
   "dependencies": {
-    "babel-core": "^6.26.3",
-    "prop-types": "^15.6.2",
-    "sweetalert2": "^7.29.2"
+    "@fortawesome/fontawesome-free": "^5.8.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-addons-create-fragment": "^15.6.2",
+    "sweetalert2": "^8.8.1"
   },
   "devDependencies": {
-    "@fortawesome/fontawesome-free": "^5.5.0",
-    "babel-cli": "6.26.0",
-    "babel-eslint": "8.2.6",
-    "babel-loader": "7.1.5",
-    "babel-plugin-transform-object-rest-spread": "6.26.0",
-    "babel-preset-env": "1.7.0",
-    "babel-preset-react": "6.24.1",
-    "eslint": "5.3.0",
-    "eslint-config-google": "0.9.1",
-    "eslint-loader": "2.1.0",
-    "eslint-plugin-react": "7.11.1",
-    "prop-types": "latest",
-    "react": "^16.4.2",
-    "react-addons-create-fragment": "^15.6.2",
-    "uglifyjs-webpack-plugin": "^1.2.7",
-    "webpack": "4.16.5",
-    "webpack-cli": "3.1.0"
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.4.3",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.5",
+    "eslint": "^5.16.0",
+    "eslint-config-google": "^0.12.0",
```

Comment:
Just did some testing and it looks like you can update everything without causing any syntax errors if you leave out `eslint-config-google` and keep it at version 0.9.1

I think it's best for organization purposes to update the security-related packages in this PR and have another cleanup/style change PR separately. It goes with our convention of trying to separate `Cleanup` tasks from `Bug Fix` tasks.

It makes the review process and git history cleaner. Also gives more of an opportunity for JS people to decide whether to follow Google's indentation standards or keep things as they have been.

For reference, these are the only changes made between `eslint-config-google` between versions 0.9.1 and 0.12 https://github.com/google/eslint-config-google/compare/0.9.1...master

Reply:
@johnsaigle I'm a little confused why this PR is difficult to review. The first commit does the changes. All the other commits were just updating the syntax with the automatic eslint command. This is following the suggestion of @driusan 

I think we're supposed to be following the eslint-config-google standard as it's always been included in the configuration file but for some reason it wasn't being in-forced like now.

I could make another PR with removing eslint-config-google and update the modules (that's also the point of this PR). Then make another PR that updates esling-config-google and apply the syntax changes. I just think it would be easy if people just reviewed the first commit because all the other commits afterwards are just running the automatic eslint fixes. I'm not doing anything manually that would break anything.